### ISSUE TITLE
Alterac Valley  Vandar / Drek

### ISSUE THREAD

**Issue by Bigepo** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#38]
Vandar / Drek HP / Damage at the moment is so unbalanced.
Drek 2shotting 440 def warrior and vandar is tanked by hunterspet...
Alliance dont have full T3 yet..

**Comment by Kaiitothewai** [comment]
Yeah, this was a big issue back in 2014 too, shame hope it gets fixed soon for you

**Comment by ivid** [comment]
Fixed, coming soon on PTR.
